src/lua/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ if (LUAJIT_LIBRARIES AND LUAJIT)
 
 	include_directories(${LUAJIT_INCLUDE_DIR})
 	add_executable(bcc-lua src/main.c bcc.o)
+	set_target_properties(bcc-lua PROPERTIES LINKER_LANGUAGE C)
```

Comment:
It's a WIP-fix, actually. Sorry I didn't make it obvious. It was picking `g++` to compile the source program, which caused `bcc-lua` to depend on `libstdc++` (when ideally it wouldn't, because we're already linking is statically).

Changing it to compile using `gcc` now parses `main.c` as C code (which is a good thing), but for some reason CMake still insists on linking `libstdc++`... So I need to figure out Step 2 to remove that dependency.